docs/content/preview/manage/backup-restore/point-in-time-recovery.md

```diff
@@ -81,8 +81,16 @@ To create a schedule and enable PITR, use the [`create_snapshot_schedule`](../..
 
 Assuming the retention target is three days, you can execute the following command to create a schedule that produces a snapshot once a day (every 1,440 minutes) and retains it for three days (4,320 minutes):
```

Comment:
```suggestion
For example, to create a schedule that produces a snapshot of a YSQL database once a day (every 1,440 minutes) and retains it for three days (4,320 minutes), you would execute the following command:
```

Reply:
Should also be clarified/made more explicit in the reference doc https://docs.yugabyte.com/preview/admin/yb-admin/#create-snapshot-schedule